contracts/modules/GuestModule.sol

```diff
@@ -29,17 +32,20 @@ contract GuestModule is
   ModuleCalls,
   ModuleCreator
 {
+  error DelegateCallNotAllowed();
```

Comment:
MultiCallUtils also adds the transaction index to the error as well, do we want to do that here too?